src/main/src/main/java/org/geoserver/data/InternationalContentHelper.java

```diff
@@ -67,6 +67,20 @@ public InternationalContentHelper(
         verify();
     }
 
+    public InternationalContentHelper(String language) {
```

Comment:
The LegendGraphicRequest gets the Locale set from the Kvp reader.
I guess it is better to stick to that one instead of adding a constructor to the InternationalContentHelper that is more bound the GetCapabilities i18n support.

Reply:
done